### PR TITLE
Better documetation

### PR DESCRIPTION
**Pull request by Infinitybeond1**
This pull req aims to make the documentation more user friendly and easier to follow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+You can run ./install.sh and ./uninstall.sh for an automated install
+
+## Build It Yourself
+To build the flatpak by yourself, run
+1. build.sh (generates build-dir)
+2. run_bash.sh and verify that the container works.
+3. export.sh (generates GJTestRepo)
+
+## Disclaimer
+PLEASE NOTE: ROBLOX PLAYER DOES NOT FUNCTION YET, DUE TO ISSUES WITH VMPROTECT
```
Comment (Soikr) [inline README.md:32]: Not true, this will work with the flatpak:

**app/org.winehq.Wine/x86_64/stable-21.08**

Comment (Infinitybeond1) [reply README.md:32]: oh sorry about that, i copied that part from the existing readme

Comment (Thelolguy1) [reply README.md:32]: Nevermind, I can handle it

**Review by Thelolguy1 — APPROVED** [review]
Looks good

**Pull request merged**